src/desktop/apps/style_guide/index.coffee

```diff
@@ -1,9 +1,7 @@
 express = require 'express'
-routes = require './routes'
 
 app = module.exports = express()
-app.set 'views', "#{__dirname}/templates"
-app.set 'view engine', 'jade'
 
-app.get '/style-guide', routes.index
-app.get '/style-guide/:section', routes.section
+app.all '/style-guide', (req, res) ->
+  res.redirect 301, "https://palette.artsy.net/"
```

Comment:
This looks good to me. Are we committed to not using https://artsy.net/style-guide in the future?

If we think we might use the same path again, we might want to roll with a 302 Temporary Redirect instead of a 301 Permanent Redirect. What do you think?

Reply:
That makes sense to me.

Reply:
This hasn't been used in years. 301 seems better because Palette is our style guide.

Reply:
(If anything, this route can just go away completely as cleanup.)

Reply:
Well, we don't want it to go away completely because it shows up in SEO results related to Artsy's branding and design. That's why I didn't delete it to begin with.

Reply:
> This hasn't been used in years. 301 seems better because Palette is our style guide and everything else is deprecated.

Sounds good to me. Thanks for the context @damassi :+1: